src/posts/2020-10-15-no-mouse-days.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Enabling keyboard culture with No Mouse Days
+description: Tips on ditching your mouse and creating more accessible experiences.
+category: How-to
+author: Marcy Sutton
+date: 2020-10-15
+further_reading:
+  - title: "The #NoMouse Challenge"
+    url: https://nomouse.org/
+    source: The DO-IT Center at University of Washington
+  - title: No Mouse Days on npm
+    url: https://www.npmjs.com/package/no-mouse-days
+    source: Marcy Sutton
+    year: 2019
+  - title: No Mouse Days on GitHub
+    url: https://github.com/marcysutton/no-mouse-days/
+    source: Marcy Sutton
+    year: 2019
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - howto
+---
+
+Wouldn’t it be great if designers and developers tested their digital experiences for keyboard accessibility–and by extension, screen reader accessibility–on the regular? That’s where the #NoMouse challenge and No Mouse Days come in.
```

Comment:
I love me a good emdash, but could we break this into two sentences? I'd like to try and keep the reading level lower.

Reply:
Sure thing!